### PR TITLE
Feature/enable config channel

### PR DESCRIPTION
**Pull request by amaredeus**
3. Enable poap config via channel flow
4. Add poap config status command to display list of authorized users and roles
5. Rename `/poap schedule` -> `/poap mint`

### CODE REVIEW

File: src/app/service/poap/config/StatusPOAP.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	const roleDescription = 'List of authorized roles that can use the POAP commands.';
+	const authorizedRolesMsg: MessageOptionsSlash | MessageOptions = ServiceUtils.generateEmbedFieldsMessage(isDmOn, authorizedRoles, roleTitle, roleDescription);
+	
+	if (isDmOn) {
+		await guildMember.send(authorizedUsersMsg as MessageOptions);
+		await guildMember.send(authorizedRolesMsg as MessageOptions);
+	} else {
+		await ctx.sendFollowUp(authorizedUsersMsg as MessageOptionsSlash);
+		await ctx.sendFollowUp(authorizedRolesMsg as MessageOptionsSlash);
+	}
```
Comment (nonsensecodes) [inline src/app/service/poap/config/StatusPOAP.ts:69]: Are these supposed to be doubled up?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Two comments. One grammar related, one for clarification.

File: src/app/service/poap/config/StatusPOAP.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/service/poap/config/StatusPOAP.ts:69]: It's very subtle but lines 67 and 68 are two different executions. The first sends a message for authorized **users** the second sends a message for authorized **roles**

Comment (nonsensecodes) [reply src/app/service/poap/config/StatusPOAP.ts:69]: Ah yes, okay, I see. Thank you for the clarification.

**Review by nonsensecodes — APPROVED** [review]
Awesome, thank you @amaredeus!

**Pull request merged**